src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1406,9 +1406,7 @@ class S4Importer final : public IParkImporter
 
         dst->state = static_cast<PeepState>(src->state);
         dst->sub_state = src->sub_state;
-        dst->next_x = src->next_x;
-        dst->next_y = src->next_y;
-        dst->next_z = src->next_z / 2;
+        dst->NextLoc = { src->next_x, src->next_y, src->next_z * 4 };
```

Comment:
Is there a RCT1_COORDS_Z_STEP constexpr? Could you create one if it doesn't exist yet?